packages/ceramic-core/src/anchor/anchor-service-factory.ts

```diff
@@ -20,6 +20,7 @@ export class AnchorServiceFactory {
      * Get AnchorService instance by type
      */
     public get(): AnchorService {
+        console.log('anc', this._config.anchorServiceUrl)
```

Comment:
do we need this `console.log`?

Reply:
Sure as hell no. Sorry for that. And thank you for finding this.

Reply:
awesome! no worries! 😊